Source/WebCore/page/Page.h

```diff
@@ -1311,6 +1315,8 @@ class Page : public Supplementable<Page>, public CanMakeWeakPtr<Page> {
     UniqueRef<StorageProvider> m_storageProvider;
     UniqueRef<ModelPlayerProvider> m_modelPlayerProvider;
 
+    KeyboardScrollingAnimator* m_currentKeyboardScrollingAnimator { nullptr };
```

Comment:
We don't use raw pointers in member variables for security reasons, so this would need to be a `RefPtr` or `WeakPtr`. If we can, we should avoid having to store it at all.